cves/CVE-2011-1088.yml

```diff
@@ -105,9 +115,13 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: |
+    Fixing this involved adding new unit tests as well to prevent this from happening
+    in the future. Multiple new regression tests were also added to prevent the
+    introduced bugs from reappearing in the future. The original code was partially,
+    but not fully tested.
```

Comment:
How fully tested was the original code after the new tests were added? Do the new tests cover all of the vulnerability, or only most of it?

Reply:
Added in some more info about the tests.